my_list.rb

```diff
@@ -0,0 +1,25 @@
+class MyList
+  attr_reader :list
+
+  include MyEnumerable
+
+  def initialize(*args)
+    @list = args
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
+end
+
+list = MyList.new(1, 2, 3, 4)
+
+list.all? { |e| e < 5 }
+
+list.all? { |e| e > 5 }
+
+list.any? { |e| e == 2 }
+
+list.any? { |e| e == 5 }
+
+list.filter(&:even?)
```

Comment:
- [ ] [REQUIRED] Great work you have done so far. I noticed you have not required the file `myenum.rb` in your `my_list` class. 
at the top of the `my_list.rb` file, include the code below: 
```
require_relative './myenum.rb'
```
See below the error I am getting when I run your project. 
![image](https://user-images.githubusercontent.com/84156714/215099198-9c0b53d0-b6a2-4111-a242-25a44526de7e.png)

Reply:
@bobb-Rob Thanks for finding error